components/ai/code/ai-bbe/create-images-with-dall-e.md

```diff
@@ -1,4 +1,4 @@
 ---
 title: 'Create images with DALL-E and Stable Diffusion'
-description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. Ballerina’s powerful standard library makes uploading, downloading and processing images a breeze."
+description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. Ballerina’s powerful library makes uploading, downloading and processing images a breeze."
```

Comment:
```suggestion
description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. The powerful Ballerina libray makes uploading, downloading and processing images a breeze."
```

Reply:
IMO Ballerina library going together is better as it is consistent and give correct context. Check and fix all places.

Reply:
Fixed.